test/performance/odp_l2fwd_run.sh

```diff
@@ -71,17 +71,10 @@ run_l2fwd()
 
 	GEN_PID=$!
 
-	# this just turns off output buffering so that you still get periodic
-	# output while piping to tee, as long as stdbuf is available.
-	if [ "$(which stdbuf)" != "" ]; then
-		STDBUF="stdbuf -o 0"
-	else
-		STDBUF=
-	fi
 	LOG=odp_l2fwd_tmp.log
 
 	# Max 2 workers
-	$STDBUF odp_l2fwd${EXEEXT} -i $IF1,$IF2 -m 0 -t 5 -c 2 | tee $LOG
```

Comment:
The commit message says: "Using stdbuf is only necessary when something in the output is being waited on, with e.g. tail -f. That's not the case here, so remove the unnecessary usage."

I am not sure the use of stdbuf here is unnecessary. When running the script in an interactive shell, the stfbuf causes the pps printouts to appear as soons as available, not after the scrip has run. So there is some benefit there. I am not sure if it outweighs the trouble though, so I am not sure what to think of this commit.

Reply:
In some test scripts the script starts a task to the background and then waits for something in the background task's output. In those cases stdbuf is necessary for the script to work, but that is not the case here, hence "unnecessary". But anyway, commit comment reworded in v7.